python/chronos/src/bigdl/chronos/metric/__init__.py

```diff
@@ -13,3 +13,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+from .forecast_metrics import Evaluator
```

Comment:
we have a tricky issue here since other metrics may be used here for anomaly detection or other tasks (e.g., classification) in future. We may still to this and make `Evaluator` support more metrics in future.